tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1240,4 +1245,200 @@ public function checkUserInformation(array $expectedValue, array  $actualValue):
 			}
 		}
 	}
+
+	/**
+	 * @When user :user tries to get information of user :ofUser using Graph API
+	 *
+	 * @param string $user
+	 * @param string $ofUser
+	 *
+	 * @return void
+	 */
+	public function userTriesToGetInformationOfUser($user, $ofUser): void {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		$response = GraphHelper::getUser(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$credentials['username'],
+			$credentials['password'],
+			$ofUser
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @When user :user tries to get all user using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsAllUserUsingTheGraphApi(string $user) {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		$response = GraphHelper::getUsers(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$credentials['username'],
+			$credentials['password'],
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @Then the API response should contain all user with following information:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @throws Exception
+	 * @return void
+	 */
+	public function theApiResponseShouldContainAllUserWithFollowingInformation(TableNode $table): void {
+		$values = $table->getHash();
+		$apiResponse = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse())['value'];
+		foreach ($values as $expectedValue) {
+			$found = false;
+			foreach ($apiResponse as $key => $actualResponseValue) {
+				if ($expectedValue["displayName"] === $actualResponseValue["displayName"]) {
+					$found = true;
+					$this->checkUserInformation($expectedValue, $actualResponseValue);
+					unset($apiResponse[$key]);
+					break;
+				}
+			}
+			if (!$found) {
+				throw new Exception('User ' . $expectedValue["displayName"] . ' could not be found in the response.');
+			}
+		}
+	}
+
+	/**
+	 * @param string $user
+	 * @param string|null $ofUser
+	 *
+	 * @return ResponseInterface
+	 * @throws JsonException
+	 * @throws GuzzleException
+	 */
+	public function retrieveUserInformationAlongWithDriveUsingGraphApi(
+		string $user,
+		?string $ofUser = null
+	):ResponseInterface {
+		if ($ofUser === null) {
+			$ofUser = $user;
+		}
```

Comment:
```suggestion
		$user = $user ?? $byUser;
```
I prefer `null coalescing operator`